pages/your_tasks/zenodo_doi.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Adding a DOI in your code releases with Zenodo
+search_exclude: true
+description: Covers how to obtain a DOI via Zenodo for your code releases 
+contributors: ["Shoaib Sufi"] 
+page_id: zenodo_doi
+related_pages: []
+---
+## How can I get a Digital Object Identifier (DOI) for my code release?
+
+### Description 
+Obtaining a DOI (a type of persistent identifier) for your software has become [increasingly popular][datacite-doi-software]. So, how do you obtain a DOI (a type of persistent identifier) to refer to your software releases? And what are the benefits; this could be because you or another user of the code would like to refer to it in a publication or you would like a global repository independent way of referring to it. 
+
+### Considerations 
+* You use [GitHub][github] as your software repository
+* You make software releases (e.g. using [semantic versioning][semantic-versioning])
+* You would like a DOI for your software releases
+* You use or are happy to make an account on the [Zenodo][zenodo] archive 
+
+### Solutions 
+
+* Link your GitHub repository to Zenodo
+	* Go to Zenodo.
+	* If you don’t have an account, make an account.
+	* Navigate to your [Zenodo my account page](https://zenodo.org/account/settings/profile)
+	* Click on the GitHub option, Click on Connect
+	* Authorise Zenodo to access your GitHub account
+	* Choose which repository you would like to create a DOI for under the repositories sections
+		* You may need to scroll down to find the repository if you have access to many repositories
+
+* Create a GitHub release for the repository you have enabled
+	* Go to your repository
+	* Click on releases and then on Draft a new release
+	* Remember to use a version number (semantic versioning is the common standard but there are other schemes such as [CalVer][calver] which is date based)
+	
+* Zenodo will automatically archive this release:
+	* Once your GitHub repository is linked to Zenodo, any new release you publish on GitHub will be archived by Zenodo
+	* Zenodo will automatically issue a DOI for your release
+
+* Get the DOI:
+	* After the GitHub release, go to Zenodo, where you will see your repository archived with a DOI assigned to it
```

Comment:
We should state how to get the badge. 
In "details", click on the badge:
![image](https://github.com/user-attachments/assets/616d5f3d-e723-4065-9dc3-a4faeeae8c0c)
Then copy the markdown code in your readme. It should show as follows:
![image](https://github.com/user-attachments/assets/0f8e366f-9a11-40d7-a75b-11ef9fa50336)